### ISSUE TITLE
Snap does not work at all in Jammy

### ISSUE THREAD

**Issue by jpseara** [ogra1/zoom-snap#112]
Application does not even open in Jammy.

Easy to reproduce: Just spin up an Ubuntu 22.04 Desktop VM, then run `sudo snap install zoom-client`.

Installation succeeds but the application never opens once one runs `zoom-client`.

Attached goes the log files. There are also some crash dumps in the directory, but they are of size 0.

```
jpseara@JP:~/Desktop$ sudo snap info zoom-client
name:      zoom-client
summary:   ZOOM Cloud Meetings
publisher: Oliver Grawert (ogra)
store-url: https://snapcraft.io/zoom-client
contact:   ogra@ubuntu.com?subject=zoom-client
license:   unset
description: |
  Video conferencing with real-time messaging and content sharing
  
  https://zoom.us provides simplified video conferencing, whiteboard sharing
  and messaging across any device. This is an unofficial re-pack of the
  debian
  package provided by zoom.us
commands:
  - zoom-client
snap-id:      76rrD7USwCJrZgepbRk7UdFEWON3tVKX
tracking:     latest/stable
refresh-date: 2 days ago, at 16:05 WEST
channels:
  latest/stable:    5.10.3.2778 2022-04-21 (175) 369MB -
  latest/candidate: 5.10.3.2778 2022-04-21 (175) 369MB -
  latest/beta:      5.10.3.2778 2022-04-21 (175) 369MB -
  latest/edge:      5.10.3.2778 2022-04-20 (175) 369MB -
installed:          5.10.3.2778            (175) 369MB -
```

[zoom_stdout_stderr.log](https://github.com/ogra1/zoom-snap/files/8550084/zoom_stdout_stderr.log)

**Comment by jpseara** [comment]
Seems like the maintainers have removed latest version from the "stable" branch, which means my client automatically downgraded and is now working again.

`sudo snap refresh zoom-client` to fix it in 22.04.

As I assume the maintainers will not put out again a version without testing it for 22.04, I am closing this case for now.

**Comment by Fuseteam** [comment]
it....works? cool!
